JASP-Desktop/JASP-Desktop.pro

```diff
@@ -205,24 +209,29 @@ unix {
   GETTEXT_LOCATION = $$(GETTEXT_PATH)
   isEmpty(GETTEXT_LOCATION): GETTEXT_LOCATION=/usr/local/bin
 
-  EXTENDED_PATH = $$(PATH):$$GETTEXT_LOCATION:$$_R_HOME
+  EXTENDED_PATH = $$(PATH):$$GETTEXT_LOCATION:$$_R_HOME:$$(QTDIR)/bin
 
   delres.commands += rm -rf $$RESOURCES_DESTINATION;
   copyres.commands += $(MKDIR) $$RESOURCES_DESTINATION ;
   copyres.commands += cp -R $$RESOURCES_PATH/* $$RESOURCES_DESTINATION ;
 
   $$GENERATE_LANGUAGE_FILES {
+  
     maketranslations.commands += $$quote(echo "Generating language Files");
-	maketranslations.commands += export PATH=$$EXTENDED_PATH;
+    maketranslations.commands += export PATH=$$EXTENDED_PATH;
     maketranslations.commands += lupdate -locations none -extensions cpp,qml -recursive $$PWD/.. -ts $$SOURCES_TRANSLATIONS/jasp.po ;
-    maketranslations.commands += lupdate -locations none -extensions cpp,qml -recursive $$PWD/.. -ts $$SOURCES_TRANSLATIONS/jasp_$${LANGUAGE_CODE}.po ;
-
     #cleanup po files
 	maketranslations.commands += msgattrib --no-obsolete --no-location $$SOURCES_TRANSLATIONS/jasp.po -o $$SOURCES_TRANSLATIONS/jasp.po ;
-	maketranslations.commands += msgattrib --no-obsolete --no-location $$SOURCES_TRANSLATIONS/jasp_$${LANGUAGE_CODE}.po -o $$SOURCES_TRANSLATIONS/jasp_$${LANGUAGE_CODE}.po ;
 
-    #Create jasp_$${LANGUAGE_CODE}.qm
-    maketranslations.commands += lrelease $$SOURCES_TRANSLATIONS/jasp_$${LANGUAGE_CODE}.po -qm $$RESOURCES_TRANSLATIONS/jasp_$${LANGUAGE_CODE}.qm ;
+    for(LANGUAGE_CODE, SUPPORTED_LANGUAGES) {
+       message("*********** Busy with languagecode $${LANGUAGE_CODE} ******************")
```

Comment:
This should be something like `$$quote(echo "Generating language Files");` otherwise it is printed when qmake is run and not when you are actually "busy with it" 

https://www.youtube.com/watch?v=oPQ3o14ksaM